SwanProjects/bin/swan_bash

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -i
+#Author Omar.Zapata@cern.ch 2021
+
+clear
+if ! [ -x "$(command -v jq)" ]; then
+  echo 'Error: jq is not installed.' >&2
+  sleep 60
+  exit 1
+fi
+if [[ $# -gt 0 ]] ; then
+    PROJECT=$1
+    PROJECT_PATH="$HOME/SWAN_projects/$PROJECT"
+    PROJECT_FILE="$PROJECT_PATH/.swanproject"
+    
+    if [ -d "$PROJECT_PATH" ]
+    then
+      STACK=`jq '.stack' $PROJECT_FILE`
+      RELEASE=`jq '.release' $PROJECT_FILE`
+      PLATFORM=`jq '.platform' $PROJECT_FILE`
+      USER_SCRIPT="$PROJECT_PATH/.userscript"
+      env -i HOME=$HOME OAUTH2_FILE=$OAUTH2_FILE \
+             OAUTH2_TOKEN=$OAUTH2_TOKEN \
+             OAUTH_INSPECTION_ENDPOINT=$OAUTH_INSPECTION_ENDPOINT \
+             PROJECT=$PROJECT \
+             PROJECT_PATH=$PROJECT_PATH PS1="$PS1" \
+             PATH="/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin:$HOME/.local/bin/" \
+             bash -c "swan_env $PROJECT $PROJECT_PATH  bash --rcfile  <(echo 'PS1=\"($PROJECT) $PS1 \"') " 
+    else
+      echo "Error: project $PROJECT_PATH doesn't exists" >&2
+      sleep 60
```

Comment:
Why this sleep?

Reply:
This is because if it fails loading bash for the project, the SwanTerminal extension will close the terminal immediately without telling the user what happened, that's why I put a sleep to give the user time to read the error.

Reply:
Does it close the terminal window completely? In the current Jupyter, if you `exit` the terminal process from the shell prompt, the window is still there.

Reply:
may to try it in jupyterhub not in lab, lets try it in lab and it closes the window.

Reply:
Ok, then please add a comment before the sleeps to explain why you need them.